src/modules.c

```diff
@@ -359,7 +359,6 @@ Obj FuncSHOW_STAT(Obj self)
 
     /* make a list of modules with crc values                              */
     modules = NEW_PLIST(T_PLIST, 2 * im);
-    SET_LEN_PLIST(modules, 2 * im);
 
     for (k = 0, im = 1; CompInitFuncs[k]; k++) {
```

Comment:
```suggestion
     for (k = 0; CompInitFuncs[k]; k++) {
```
... and then get rid of the declaration of `im`, too.